src/useClientRequest.js

```diff
@@ -57,7 +57,7 @@ function useClientRequest(query, initialOpts = {}) {
   });
 
   // arguments to fetchData override the useClientRequest arguments
-  async function fetchData(newOpts) {
+  async function fetchData(newOpts, updateResult) {
```

Comment:
What if we made `updateResult` part of the `options` object?

Reply:
@Joezo suggested the same, will move it 👍